dfdaemon/downloader/dfget/dfget.go

```diff
@@ -96,6 +99,9 @@ func (dfGetter *DFGetter) parseCommand(url string, header map[string][]string, n
 		args = append(append(args, "-s"), dfGetter.rateLimit)
 		args = append(append(args, "--totallimit"), dfGetter.rateLimit)
 	}
+	if dfGetter.supernodes != nil {
```

Comment:
It's better to check the length of `supernodes` is whether greater than zero. If it's not nil and its length is equals 0, dfget will execute failed.

Reply:
DONE.